docs/book/GLOSSARY.md

```diff
@@ -128,6 +142,10 @@ A generically understood combination of a kernel and system-level userspace inte
 
 # P
 
+## Pivot
+
+Pivot deploys the provided provider components to a Target cluster and then migrates all Cluster API resources from the Bootstrap cluster to the Target cluster.
```

Comment:
I would modify this to something along the lines of: Pivot is a process for moving the provider components and declared cluster-api resources from a source cluster to a target cluster.

The pivot process is also used for deleting a management cluster and could also be used during an upgrade of the management cluster.